File: src/main/java/aurocosh/divinefavor/common/potions/common/ModBlessings.java
package aurocosh.divinefavor.common.potions.common;

import aurocosh.divinefavor.common.potions.base.potion.ModPotion;
import aurocosh.divinefavor.common.potions.presences.*;

public class ModBlessings {
    public static ModPotion chilling_presence;
    public static ModPotion energetic_presence;
    public static ModPotion furious_presence;
    public static ModPotion industrious_presence;
    public static ModPotion manipulative_presence;
    public static ModPotion predatory_presence;
    public static ModPotion scorching_presence;
    public static ModPotion towering_presence;
    public static ModPotion warping_presence;

    public static void preInit() {
        chilling_presence = new PotionChillingPresence();
        energetic_presence = new PotionEnergeticPresence();
        furious_presence = new PotionFuriousPresence();
        industrious_presence = new PotionIndustriousPresence();
        manipulative_presence = new PotionManipulativePresence();
        predatory_presence = new PotionPredatoryPresence();
        scorching_presence = new PotionScorchingPresence();
        towering_presence = new PotionToweringPresence();
        warping_presence = new PotionWarpingPresence();
    }
}

File: src/main/kotlin/aurocosh/divinefavor/common/potions/common/ModBlessings.kt
package aurocosh.divinefavor.common.potions.common

import aurocosh.divinefavor.common.potions.base.potion.ModPotion
import aurocosh.divinefavor.common.potions.presences.*

object ModBlessings {
    lateinit var chilling_presence: ModPotion
    lateinit var energetic_presence: ModPotion
    lateinit var furious_presence: ModPotion
    lateinit var industrious_presence: ModPotion
    lateinit var manipulative_presence: ModPotion
    lateinit var predatory_presence: ModPotion
    lateinit var scorching_presence: ModPotion
    lateinit var towering_presence: ModPotion
    lateinit var warping_presence: ModPotion

    fun preInit() {
        chilling_presence = PotionChillingPresence()
        energetic_presence = PotionEnergeticPresence()
        furious_presence = PotionFuriousPresence()
        industrious_presence = PotionIndustriousPresence()
        manipulative_presence = PotionManipulativePresence()
        predatory_presence = PotionPredatoryPresence()
        scorching_presence = PotionScorchingPresence()
        towering_presence = PotionToweringPresence()
        warping_presence = PotionWarpingPresence()
    }

    fun init() {
        PotionToweringPresence.init()
    }
}
